ggml.c

```diff
@@ -20480,6 +20229,11 @@ size_t ggml_quantize_chunk(enum ggml_type type, const float * src, void * dst, i
         default:
             assert(false);
     }
+
+    GGML_ASSERT(result == nrows * row_size);
+
+    GGML_UNUSED(hist); // TODO: populate
```

Comment:
I see that the code to collect the histograms has been removed, are there any plans to reintroduce it? Otherwise, this parameter could be removed.